mne/preprocessing/xdawn.py

```diff
@@ -425,19 +425,19 @@ def _apply_epochs(self, epochs, include, exclude, event_id):
         if not epochs.preload:
             raise ValueError('Epochs must be preloaded to apply Xdawn')
 
-        picks = pick_types(epochs.info, meg=False, ref_meg=False,
-                           include=self.ch_names, exclude='bads')
+       # picks = pick_types(epochs.info, meg=False, ref_meg=False,
+                           # include=self.ch_names, exclude='bads')
```

Comment:
Also, I made them as comments, because I was checking the results. I'll
remove all the changes completely in the next commit.

On Wed, Jun 1, 2016 at 4:14 PM, Asish Panda <asishrocks95@gmail.com> wrote:

> Yes. Whats the best way to reset it back to upstream master while not
> deleting the commits? `xdawn.py`, its tests, and `__init__` file should be
> restored.
> I think `git checkout HEAD -- xdawn.py` should work.
>
> On Wed, Jun 1, 2016 at 4:01 PM, Jean-Rémi KING <notifications@github.com>
> wrote:
>
>> In mne/preprocessing/xdawn.py
>> <https://github.com/mne-tools/mne-python/pull/3245#discussion_r65337319>:
>>
>> > @@ -425,19 +425,19 @@ def _apply_epochs(self, epochs, include, exclude, event_id):
>> >          if not epochs.preload:
>> >              raise ValueError('Epochs must be preloaded to apply Xdawn')
>> >
>> > -        picks = pick_types(epochs.info, meg=False, ref_meg=False,
>> > -                           include=self.ch_names, exclude='bads')
>> > +       # picks = pick_types(epochs.info, meg=False, ref_meg=False,
>> > +                           # include=self.ch_names, exclude='bads')
>>
>> you should leave this since this class isn't modified
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/mne-tools/mne-python/pull/3245/files/f42f32786cd5d70fc2b65c3cffd9a9df06c57371..42c94d67e0bf48f5555e31a19788872c3b0a1f38#r65337319>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AGHZ4hk71dgsJsTcPd7zFOh3MOkYzujcks5qHV-agaJpZM4Ij7PZ>
>> .
>>
>
>